srcpkgs/ttyplot/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'ttyplot'
+pkgname=ttyplot
+version=1.1
+revision=1
+build_style=gnu-makefile
+hostmakedepends="ncurses-devel"
+short_desc="Realtime plotting utility for terminal with data input from stdin"
+maintainer="misuchiru03 <misuchiru03+void@gmail.com>"
+license="Unlicense"
+homepage="https://github.com/misuchiru03/ttyplot"
```

Comment:
Also pls wait for upstream to tag a new proper release instead of making your own.

Reply:
Right on.  Will do.